psi4/driver/driver_cbs.py

```diff
@@ -1332,20 +1496,26 @@ def cbs(func, label, **kwargs):
     finalenergy = 0.0
     finalgradient = core.Matrix(natom, 3)
     finalhessian = core.Matrix(3 * natom, 3 * natom)
+    hiloargs = dict()
+    if scf_alpha:
+        hiloargs["scf_alpha"] = scf_alpha
+    if corl_alpha:
+        hiloargs["corl_alpha"] = corl_alpha
```

Comment:
I think I have a plan on how to get independent `delta_alpha`. Let me look a bit further.

Reply:
I've implemented it for the first delta. If you plan to re-work the delta handling in the future anyway, perhaps it's not that important to extend it to further deltas - what would a two-delta method look like, anyway?